docs/usage/self-hosted-configuration.md

```diff
@@ -1109,11 +1109,12 @@ Defines how the report is exposed:
 
 ## repositories
 
-Elements in the `repositories` array can be an object if you wish to define more settings:
+Elements in the `repositories` array can be an object if you wish to define more settings.
+Example:
 
 ```js
```

Comment:
Is JavaScript the correct syntax highlight language? Most config examples use JSON.